src/main/templates/io/lettuce/core/api/BaseRedisCommands.java

```diff
@@ -65,6 +65,14 @@ public interface BaseRedisCommands<K, V> {
      */
     Map<K, Long> pubsubNumsub(K... channels);
 
+    /**
+     * Returns the number of subscribers (not counting clients subscribed to patterns) for the specified shard channels.
+     *
+     * @param channels channel keys.
+     * @return array-reply a list of channels and number of subscribers for every channel.
+     */
+    Map<K, Long> pubsubShardNumsub(K... channels);
```

Comment:
Nit: Add `@since 7.0` and propagate that change into the generated commands.

Reply:
Yep, sure, done